source/authentication.txt

```diff
@@ -0,0 +1,285 @@
+
+.. _arch-center-authentication-recs:
+
+========================================
+Guidance for {+service+} Authentication
+========================================
+
+.. default-domain:: mongodb
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta:   
+   :description: Learn about the different authentication mechanisms that Atlas supports.
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: onecol
+
+
+Recommendations for {+service+} Authentication
+--------------------------------------------------------------------------
+
+|service-fullname| supports a variety of authentication methods to
+ensure robust security. |service| requires all users to authenticate to 
+access the {+atlas-ui+}, |service| databases, and  the {+atlas-admin-api+}. 
+
+.. note:: 
+
+   In this context, a "user" can be a human or an application. We refer to 
+   human users as "Workforce Identity" and applications as 
+   "Workload Identity".
+
+The following table provides guidance on the recommended authentication methods 
+for staging and production ("upper") environments, and other methods you might 
+use in development and test ("lower") environments. 
+
+.. list-table:: 
+   :widths: 10 13 40 37
+   :header-rows: 1
+
+   * - Resource
+     - User 
+     - Recommendations for Production
+     - Options for Dev and Test
+
+   * - Atlas UI 
+     - Workforce (Human)
+     - | Use :ref:`Federated Auth <arch-center-federated_auth>` with an SAML 2.0 
+         identity provider, such as Okta, Microsoft Entra ID, or Ping Identity. 
+       | -- *or* --
+       | Use :ref:`Multi-factor Authentication (MFA) <arch-center-mfa>`.
+       | -- *and* --
+       | Use IP access restrictions.
```

Comment:
question about the relationships here. It's not clear to me if you should also use IP access restrictions with federated auth. Does the "and" apply to both "or" clauses, or only the second one?